core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -366,14 +366,10 @@ public Resolution resolve(AlluxioURI uri) throws InvalidPathException {
       if (mountPoint != null) {
         MountInfo info = mState.getMountTable().get(mountPoint);
         AlluxioURI ufsUri = info.getUfsUri();
+        AlluxioURI resolvedUri = PathUtils.resolveUri(ufsUri, path.substring(mountPoint.length()));
         UfsManager.UfsClient ufsClient;
-        AlluxioURI resolvedUri;
         try {
           ufsClient = mUfsManager.get(info.getMountId());
-          try (CloseableResource<UnderFileSystem> ufsResource = ufsClient.acquireUfsResource()) {
-            UnderFileSystem ufs = ufsResource.get();
-            resolvedUri = ufs.resolveUri(ufsUri, path.substring(mountPoint.length()));
-          }
```

Comment:
Based on the UFS.resolveUri() implementation, there's no need to guard this block using try-with-resource and acquiring a UFS resource.